lib/logstash/filters/mutate.rb

```diff
@@ -320,21 +324,32 @@ def convert_string(value)
     value.to_s.force_encoding(Encoding::UTF_8)
   end
 
+  def convert_boolean(value)
+    return true if value =~ TRUE_REGEX
+    return false if value.empty? || value =~ FALSE_REGEX
+    @logger.warn("Failed to convert #{value} into boolean.")
+    value
+  end
+
   def convert_integer(value)
     return 1 if value == true
     return 0 if value == false
-    value.to_i
+    value.squeeze(MUC_CP).tr(MUC_CP, MUC_UP).to_i
   end
 
   def convert_float(value)
-    value.to_f
+    value.squeeze(MUC_CP).tr(MUC_CP, MUC_UP).to_f
   end
 
-  def convert_boolean(value)
-    return true if value =~ TRUE_REGEX
-    return false if value.empty? || value =~ FALSE_REGEX
-    @logger.warn("Failed to convert #{value} into boolean.")
-    value
+  def convert_integer_eu(value)
+    return 1 if value == true
+    return 0 if value == false
+    value.squeeze(MUC_CP).tr(MUC_CP.reverse, MUC_UP).to_i
```

Comment:
is converting to `0` and `1` from `false` and `true` a documented/historical  behaviour or a new feature introduced here?